.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -33,4 +33,4 @@ Add any other context about the problem here.
 
 You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)
 
-Feel free to check out other cool repositories of Eddie Jaoude Community [here](https://github.com/EddieJaoudeCommunity)
+Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)
```

Comment:
Maybe here should be only `EddieHub` 🤔

Reply:
It name of the community wasn't changed to EddieHub but EddieHub community, that was why I did like that.

Reply:
@eddiejaoude, which of the two you think is better?

Reply:
I've approved the PR @chryzcodez, but let's wait for @eddiejaoude feedback on this topic before merging😃

Reply:
Thanks all 👍 . I think `EddieHub` might be cleaner. I will merge this, as it still fixes the link, but if a new PR could be created to just use `EddieHub`